### PR TITLE
maple-font: new, 6.4

### PR DESCRIPTION
**Pull request by CAB233**
Topic Description
-----------------

- maple-font: new, 6.4

Package(s) Affected
-------------------

- maple-font: 6.4

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [ ] AMD64 `amd64`
- [ ] AArch64 `arm64`
- [ ] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

Architectural progress for secondary ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: desktop-fonts/maple-font/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+abinfo "Installing font file..."
+
+install -Dm644 "$SRCDIR"/*.ttf \
+    -t "$PKGDIR"/usr/share/fonts/TTF
+
+install -Dm644 "$SRCDIR"/*.otf \
+    -t "$PKGDIR"/usr/share/fonts/OTF
+
+install -Dm644 "$SRCDIR"/*.woff2 \
+    -t "$PKGDIR"/usr/share/fonts/WOFF2
```
Comment (MingcongBai) [inline desktop-fonts/maple-font/autobuild/build]: Missing newline at EOF.

File: desktop-fonts/maple-font/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,10 @@
+abinfo "Installing font file..."
+
+install -Dm644 "$SRCDIR"/*.ttf \
```
Comment (MingcongBai) [inline desktop-fonts/maple-font/autobuild/build]: Verbosity here, please see the [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/).

File: desktop-fonts/maple-font/autobuild/build [hunk 3]
```diff
@@ -0,0 +1,10 @@
+abinfo "Installing font file..."
+
```
Comment (MingcongBai) [inline desktop-fonts/maple-font/autobuild/build]: Why all the blank lines, seems a waste of space.

File: desktop-fonts/maple-font/autobuild/defines [hunk 4]
```diff
@@ -0,0 +1,6 @@
+PKGNAME="maple-font"
+PKGDES="Open source monospace font with round corner, ligatures and Nerd-Font for IDE and command line."
```
Comment (MingcongBai) [inline desktop-fonts/maple-font/autobuild/defines]: Monospace font with round corner, ligatures, and nerd-font integration

Comment (MingcongBai) [reply desktop-fonts/maple-font/autobuild/defines]: Note: No period points and no unhelpful advertisement.

Again, see the [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/).

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: desktop-fonts/maple-font/autobuild/build [hunk 5]
```diff
@@ -0,0 +1,5 @@
+abinfo "Installing font file..."
+for ext in ttf otf woff2; do
+    install -Dvm644 "$SRCDIR"/*."$ext" \
+        -t "$PKGDIR"/usr/share/fonts/${ext^^}
```
Comment (MingcongBai) [inline desktop-fonts/maple-font/autobuild/build:4]: Smart.

**Review by MingcongBai — CHANGES_REQUESTED** [review]
Code LGTM. Please squash your commits.

**Review by MingcongBai — APPROVED** [review]

**Review by OriginCode — APPROVED** [review]

**Pull request merged**